funnel/templates/updates.html.jinja2

```diff
@@ -2,6 +2,7 @@
 {% set title_suffix = project.title %}
 {% from "baseframe/forms.html.jinja2" import renderfield, widgetscripts %}
 {%- from "macros.html.jinja2" import project_header, updates_template, useravatar %}
+{%- from "js/update.js" import updates_template %}
```

Comment:
Jinja2 JavaScript templates must be named `.js.jinja2`. This template seems to be a mix of HTML and JS, so I suggest splitting it up into separate template files and importing as needed.